<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BR47O9U" failures="0" tests="2" name="dineshpackageinterview.Depensonfails" time="2.016" errors="1" timestamp="2023-09-22T17:13:17 EDT" skipped="0">
  <testcase classname="dineshpackageinterview.Depensonfails" name="test" time="0.001"/>
  <testcase classname="dineshpackageinterview.Depensonfails" name="parenttest" time="2.015">
    <error message="Method dineshpackageinterview.Depensonfails.parenttest() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method dineshpackageinterview.Depensonfails.parenttest() didn't finish within the time-out 2000
at java.base@17.0.8/java.lang.Thread.sleep(Native Method)
at app//dineshpackageinterview.Depensonfails.parenttest(Depensonfails.java:11)
at java.base@17.0.8/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.8/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.8/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.8/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.8/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.8/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.8/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.8/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- parenttest -->
</testsuite> <!-- dineshpackageinterview.Depensonfails -->
